src/Services/StripePaymentService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Gcd\Scaffold\Payments\Services;
+
+class StripePaymentService implements PaymentServiceInterface
```

Comment:
I think we should keep Stripe outside of the base scaffold. I've made a stripe scaffold you can move this to.